flytepropeller/pkg/controller/executors/execution_context.go

```diff
@@ -4,7 +4,7 @@ import (
 	"github.com/flyteorg/flyte/flytepropeller/pkg/apis/flyteworkflow/v1alpha1"
 )
 
-// go:generate mockery -case=underscore
+//go:generate mockery-v2 --case=underscore --with-expecter --name ExecutionContext --output=mocks
```

Comment:
Notice that we were not generating the mocks before. https://github.com/flyteorg/flytepropeller/pull/600 introduced the `go:generate` directive but unfortunately there was a typo.

Also notice that I'm moving to upstream mockery (as described in https://github.com/flyteorg/flyte/pull/4937) because the legacy mock couldn't generate the code for `GetConsoleURL` due to the fact that it was coming to `ExecutionContext` via other interface.